travis/check_tags.py

```diff
@@ -0,0 +1,61 @@
+import re
+import requests
+
+
+def get_errors_msgs_commits(travis_repo_slug, travis_pull_request_number, travis_branch, version, token):
+    development_tags = [':memo:', ':fire:', ':fire_engine:', ':tv:', ':lock:', ':bath:', ':green_heart:', ':cat:', ':bomb:']
+    release_tags = [':tada:', ':zap:', ':sparkles:', ':rainbow:', ':ambulance:', ':heart_eyes:', ':cherries:', ':book:', ':euro:', ':handshake:', ':shield:', ':arrow_up:', ':arrow_down:', ':x:', ':sos:', ':peace_symbol:']
+    verion_tags_dict = {'zero': '0', 'one': '1', 'two': '2', 'three': '3', 'four': '4', 'five': '5', 'six': '6', 'seven': '7', 'eight': '8', 'nine': '9'}
+    verion_tags = [':zero:', ':one:', ':two:', ':three:', ':four:', ':five:', ':six:', ':seven:', ':eight:', ':nine:']
+    requirements_tags_of_version = [':x:', ':arrow_up:', ':arrow_down:', ':sos:', ':tada:']
+
+    is_addons_dev = re.search(r'addons-dev', str(travis_repo_slug))
+    symbol_in_branch = re.search(r'-', str(travis_branch))
+
+    #GET /repos/:owner/:repo/pulls/:pull_number/commits
+    #See API Github: https://developer.github.com/v3/repos/commits/#list-commits-on-a-repository
+    real_errors = {}
+    if travis_pull_request_number:
+        # GET / repos /: owner /:repo / commits
+        commits = requests.get('https://api.github.com/repos/%s/pulls/%s/commits' % (str(travis_repo_slug), str(travis_pull_request_number)), headers={'Authorization': token}).json()
+        for commit in commits:
+            commit = commit.get('commit').get('message')
+            print('Commit: %s' % commit)
+            match = re.findall(r'(:\w+:)', commit)
+            if is_addons_dev or symbol_in_branch:
+                if any(tag in match for tag in development_tags):
+                    continue
+                else:
+                    error = {commit: 'For dev branches must be Development tags!'}
+                    real_errors.update(error)
+                    continue
+            else:
+                if any(tag in match for tag in requirements_tags_of_version):
+                    if any(tag in match for tag in verion_tags):
+                        list_digits = [x for x in commit.split(':') if x in verion_tags_dict.keys()]
+                        version_in_commit = ''
+                        for digit in list_digits:
+                            version_in_commit += verion_tags_dict.get(digit)
+                        if version.replace(".", "") != version_in_commit:
+                            error = {commit: 'Version in commit is wrong!'}
+                            real_errors.update(error)
+                            continue
+                        index_requirements_tags_version = [match.index(i) for i in match if i in requirements_tags_of_version]
+                        index_verion_tags = [match.index(i) for i in match if i in verion_tags]
+                        if index_requirements_tags_version[-1] < index_verion_tags[0]:
+                            continue
```

Comment:
Too much indents. Whenever you have
```
if X:
    # something A
    # something A
    # something A
    # something A
    # something A
    # something A
    # something A
    # something A
    # something A
    # something A
    # something A
    # something A
    # something A
    # something A
    # something A
    # something A
    # something A
    # something A
    # something A
    # something A
    # something A
    # something A
    # something A
    # something A
    # something A
    # something A
    # something A
    continiue
else:
    # something B
    continiue

```
You can use
```
if not X:
    # something B
    continiue
# something A
# something A
# something A
# something A
# something A
# something A
# something A
# something A
# something A
# something A
# something A
# something A
# something A
# something A
# something A
# something A
# something A
# something A
# something A
# something A
# something A
# something A
# something A
# something A
# something A
# something A
# something A
continiue

```

Reply:
+